aten/src/ATen/native/Activation.cpp

```diff
@@ -606,6 +606,7 @@ Tensor& rrelu_with_noise_out_cpu(const Tensor& self,
     bool training,
     c10::optional<Generator> generator,
     Tensor& output) {
+  TORCH_CHECK(self.sym_sizes() == noise.sym_sizes(), "noise tensor shape must match self tensor shape. Got self.shape = ", self.sym_sizes(), " noise.shape = ", noise.sym_sizes())
```

Comment:
TORCH_CHECK does not need a `;` to be compiled correctly, but usually we still add a `;` at the end